itou/utils/templatetags/nav.py

```diff
@@ -0,0 +1,229 @@
+from django import template
+from django.urls import reverse
+from django.utils.text import slugify
+
+
+register = template.Library()
+
+
+class NavItem:
+    active = False
+
+    def __init__(
+        self,
+        *,
+        icon=None,
+        label,
+        target,
+        active_view_names,
+        matomo_event_category=None,
+        matomo_event_name=None,
+        matomo_event_option=None,
+    ):
+        self.icon = icon
+        self.label = label
+        self.target = target
+        self.active_view_names = active_view_names
+        self.matomo_event_category = matomo_event_category
+        self.matomo_event_name = matomo_event_name
+        self.matomo_event_option = matomo_event_option
```

Comment:
Pas super fan des attributs non listé dans les classes, surtout que là c'est utilisé comme API dans `NavGroup()`
```suggestion
        self.matomo_event_option = matomo_event_option
        self.active = None
```

Reply:
Je l’avais listé directement au niveau de la classe. Je viens de le déplacer dans `__init__` pour qu’il soit avec les autres. :shrug: 
https://github.com/gip-inclusion/les-emplois/pull/4483/files/5f3ae5bd2000ec4e33908bf500fb1e09af2ad1d7#diff-4ae4b0a18514f6ac8a6752236a895396b5745f7617ef7eed2087817d3810f8d7R10